api/src/main/java/com/lifegraph/team20/controller/LifeGraphsContorller.java

```diff
@@ -21,35 +14,28 @@
 import com.lifegraph.team20.models.Content;
 import com.lifegraph.team20.models.LifeGraph;
 import com.lifegraph.team20.models.UserData;
+import com.lifegraph.team20.repository.LifeGraphsRepository;
 import com.lifegraph.team20.service.ContentService;
 
-
 @CrossOrigin(origins = "*", maxAge = 3600)
 @RestController
 @RequestMapping("/life-graphs")
 public class LifeGraphsContorller {
-@RequestMapping(value = "/search", method = RequestMethod.GET)
-	public List<UserData> userData() {
-		List<UserData> userDatas = setUserData();
-		return userDatas;
-	}
 
 	@Autowired
+	LifeGraphsRepository lifeGraphsRepository;
```

Comment:
```suggestion
	private LifeGraphsRepository lifeGraphsRepository;
```
以降はアクセス修飾子に関する指摘しません。